numba/unicode.py

```diff
@@ -246,8 +249,10 @@ def _kind_to_byte_width(kind):
         return 2
     elif kind == PY_UNICODE_4BYTE_KIND:
         return 4
+    elif kind == PY_UNICODE_WCHAR_KIND:
+        raise AssertionError("PY_UNICODE_WCHAR_KIND string encountered")
     else:
-        return 4    # FIXME: wchar
+        raise AssertionError("Unexpected unicode encoding encounter")
```

Comment:
s/encounter/encountered/ ? however, as this is unreachable, it can probably be left!

Reply:
fixed